app/assets/javascripts/visualizations/highvis/bar.coffee

```diff
@@ -110,12 +110,13 @@ $ ->
 
       drawControls: ->
         super()
-        @drawGroupControls()
-        @drawYAxisControls()
+        @drawGroupControls(data.textFields)
+        @drawYAxisControls('Y Axis', globals.configs.fieldSelection,
```

Comment:
I'm assuming there's more than one place you call `drawYAxisControls` and for whatever reason the label is not "Y Axis" somewhere else?  Otherwise it seems strange to pass that as an argument :smile: